src/main.ts

```diff
@@ -313,6 +317,7 @@ export function makeUtils(options: Options): Utils {
     ...makeTimestampMethods(options, longs),
     ...longs,
     ...makeComparisonUtils(),
+    ...makeNiceGrpcServerStreamingMethodResult(),
```

Comment:
I needed to introduce a type that would be conditionally rendered once per file only when used. Could not find a better way than adding it to the global `Utils` object.